GitSearch/Models/ColorScheme.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ColorScheme.swift
+//  GitSearch
+//
+//  Created by Vldmr on 9/9/19.
+//  Copyright © 2019 mudritskiy. All rights reserved.
+//
+
+import UIKit
+
+extension UIColor {
+
+    static let mainTint = #colorLiteral(red: 0.9647058824, green: 0.9647058824, blue: 0.9647058824, alpha: 1)
+    static let secondaryTint = #colorLiteral(red: 0.9215686275, green: 0.9215686275, blue: 0.9215686275, alpha: 1)
+    
+    static let mainTitle = #colorLiteral(red: 0.9450980392, green: 0.4078431373, blue: 0.3450980392, alpha: 1)
+    static let secondaryTitle = #colorLiteral(red: 0.4235294118, green: 0.368627451, blue: 0.3215686275, alpha: 1)
+    
+    static let elementTint1 = #colorLiteral(red: 0.4274509804, green: 0.7176470588, blue: 0.8509803922, alpha: 1)
```

Comment:
Its really bad naming like var1, var2, var3 etc. It does not convey any meaning how all these variables are different.

Reply:
No changes?

Reply:
I really have no idea how to name it, I just save thing colors from other pattern to use forward